src/filters/time/time.constants.ts

```diff
@@ -1,49 +1,44 @@
-const PM = 'pm|p.m.';
-const AM = 'am|a.m.';
-const MERIDIEM = `( )?(${AM}|${PM})`;
-const FILLER_WORDS = '(at|by) ';
-const HOUR_PART = '([0-1]?[0-9]|2[0-3])';
-const MINUTE_PART = '(:[0-5][0-9])';
-
+const AM = 'am|a.m.|am.';
+const PM = 'pm|p.m.|pm.';
+const MERIDIEM = `(${AM}|${PM})`;
+const FILLER_WORDS = '(at|by)';
+const HOUR_PART = '([0-9]{1,2})';
+const DIVIDER = '(:)';
+const MINUTE_PART = '([0-9]{1,2})';
```

Comment:
I have logic in JS to reject anything over 59, it is easier to do it that way, trust me. I used the old way and it added lots complexity. At the end I was like okay, match xx:xx and then we can use JS to reject the wrong values.

e.g. `20 mins to 24` should this be accepted? we don't accept 24, but technically it is okay because the value is 23:40. With this new regex it will be easier to handle